src/content/components/landing-page/LandingIntro.astro

```diff
@@ -1,90 +1,142 @@
 ---
-import { Code, TabItem, Tabs } from "@astrojs/starlight/components";
+import { Code } from "@astrojs/starlight/components";
 import CallToAction from "@astrojs/starlight/components/CallToAction.astro";
 import pepitaExamplePath from "./pepita.txt?raw";
----
+import { Image } from 'astro:assets';
+import wollokDiagramDark from '/src/assets/wollok-lsp-diagram-dark.png';
 
+---
 <div>
   <div class="intro-container">
     <div class="welcome">
-      <h1>Wollok</h1>
-      <p id="wollok-intro-subtitle">Aprendiendo a programar con objetos</p>
+      <div class="titulo">
+        <h1>Wollok</h1>
+        <p id="wollok-intro-subtitle">Aprendiendo a programar con objetos</p>
+      </div>
       <div class="actions">
         <CallToAction
-          link="./getting_started/installation"
+          link="/website-wollok-ts/getting_started/installation"
           variant="primary"
           icon={{ type: "icon", name: "rocket" }}
+          attrs={{title: "Instalate Wollok en tu máquina en breves pasos siguiendo este link"}}
         >
           Instalar
         </CallToAction>
-        <CallToAction link="./documentation/language" variant="secondary">
+        <CallToAction link="/website-wollok-ts/documentation/language" variant="secondary" attrs={{title: "Si estás buscando cómo implementar un concepto o ayuda sobre los mensajes que podés mandar a un objeto, seguí este link"}}>
           Documentación
         </CallToAction>
+        <CallToAction link="https://youtu.be/AwkKZ41U6WE" variant="secondary" attrs={{title: "Acá Fer nos trae una demo Wollok en VSCode usando la versión 0.2.0"}}>
+          Tour
+        </CallToAction>
       </div>
     </div>
-    <div class="hide-mobile pepita-example">
-      <Tabs>
-        <TabItem label="Código">
+    <div class="pepita-example">
+        <div class="codigo">
           <Code code={pepitaExamplePath} lang="wollok" />
-        </TabItem>
-        <TabItem label="Diagrama"> imagennn </TabItem>
-      </Tabs>
+        </div>
+        <div class="diagrama">
+          <Image src={wollokDiagramDark} id="diagramaImagen" alt="El diagrama dinámico permite visualizar el estado de tus objetos en tiempo real."/>
+        </div>
     </div>
   </div>
 </div>
 
+<script>
+// Hack para poder setear la imagen del diagrama
+const setTheme = () => {
+  const htmlTag = document.getElementsByTagName('html')[0]
+  const theme = htmlTag?.getAttribute('data-theme') ?? 'dark'
+  const diagramaImg = document.getElementById('diagramaImagen')
+  diagramaImg?.setAttribute('src', `/src/assets/wollok-lsp-diagram-${theme}.png`)
+}
+
+window.onload = setTheme
+window.onchange = setTheme
+</script>
```

Comment:
Cuando buildeo la pagina esto me esta fallando 😢 (o sea corriendolo con `npm run build && npm run preview` en vez de `npm run dev`).

Entiendo que es porque Astro procesa las imagenes cuando se buildea pero no cuando corre el servidor en "modo dev"

[La docu de Astro](https://docs.astro.build/en/guides/images/#images-in-astro-files) mencionaba un metodo parecido a este pero usando la tag `<img>` en vez de `<Image>` pero no lo pude hacer andar.

Arme este PR con la forma que lo logre hacer funcionar https://github.com/uqbar-project/website-wollok-ts/pull/20

Reply:
Ah soy un gil, recién entendí lo que hiciste @ivojawer ... muy groso porque después de 15 minutos llegué a la misma conclusión. 👏🏼 👏🏼 👏🏼